frontend/src/metabase/nav/containers/MainNavbar/SidebarItems/sortable.module.css

```diff
@@ -1,8 +0,0 @@
-:global(.react-sortable-hoc-helper.sorting) {
```

Comment:
It turned out `react-sortable-hoc` isn't used anymore so these CSS files could be deleted